### PR TITLE
*: upgrade ci

### PR DESCRIPTION
**Pull request by hawkingrei**

### CODE REVIEW

**Comment by dveeden** [comment]
From CI: `ModuleNotFoundError: No module named 'tidb_settings'`

I added a second cp to fix the CI run on Django 4 in my branch:
```diff
  cp tidb_settings.py $DJANGO_TESTS_DIR/django/tidb_settings.py
+ cp tidb_settings.py $DJANGO_TESTS_DIR/django/tests/tidb_settings.py
```

**Comment by hawkingrei** [comment]
> From CI: `ModuleNotFoundError: No module named 'tidb_settings'`
> 
> I added a second cp to fix the CI run on Django 4 in my branch:
> 
> ```diff
>   cp tidb_settings.py $DJANGO_TESTS_DIR/django/tidb_settings.py
> + cp tidb_settings.py $DJANGO_TESTS_DIR/django/tests/tidb_settings.py
> ```

Thank you!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -147,6 +149,43 @@ jobs:
...
+          sudo apt-get install -y libmemcached-dev zlib1g-dev
+
+      - name: Run tests
+        run: tox
+        env:
+          DJANGO_VERSION: 4.1.7
+
+
+  django4_tidb6:
+    name: Python ${{ matrix.python-version }} | Django 4 | TiDB 5
```
Comment (dveeden) [inline .github/workflows/ci.yml:162]:
```suggestion
    name: Python ${{ matrix.python-version }} | Django 4 | TiDB 6
```

**Comment by wd0517** [comment]
close this pr as it is stale

**Pull request closed without merging**